web/pandas/getting_started.md

```diff
@@ -1,5 +1,29 @@
 # Getting started
 
+## Try it in your browser
+
+You can try `pandas` in your browser with the following interactive shell
+without installing anything on your computer.
+
+For example you can use the following code snippet to interact with the `iris`
+dataset:
+
+```python
+import pandas as pd
+
+# load the dataset
+df = pd.read_csv('iris.csv')
+
+# display the DataFrame
+df
+```
```

Comment:
Maybe it's just me, but I find a bit confusing providing a code block with some code, and then different code in the shell. I'd personally use just one in the shell.

Reply:
Agree let's only use the injected code in the shell. This could be iterated on as we go if needed.